Description: Use Debian packaged libmimalloc-dev
Last-Update: 2022-09-23
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/build.rs
+++ b/build.rs
@@ -1,6 +1,8 @@
-use std::env;
+//use std::env;
 
 fn main() {
+    println!("cargo:rustc-link-lib=mimalloc");
+    /*
     let mut build = cc::Build::new();
 
     build.include("c_src/mimalloc/include");
@@ -47,2 +49,2 @@
     }
-}
+*/}
